src/renderer/components/ProgramChangeView.tsx

```diff
@@ -70,6 +80,23 @@ function ProgramChangeView() {
     displayList.push(key);
     pinnedAccount[key] = true;
   });
+
+  function sortFunctionDUP(a: AccountInfo, b: AccountInfo) {
+    switch (sortColumn) {
+      case 4: // count
```

Comment:
These cases should be a TypeScript enum, https://www.typescriptlang.org/docs/handbook/enums.html